lib/data_update_scripts/20201019192035_backfill_registered_for_users.rb

```diff
@@ -0,0 +1,12 @@
+module DataUpdateScripts
+  class BackfillRegisteredForUsers
+    def run
+      return if User.created_at.nil?
```

Comment:
**Question for my reviewers**: if necessary, how should I differentiate between users who were invited but _have not yet accepted the invitation (are not registered yet) and users who were invited and _have accepted the invitation_ (are, or should be considered, registered)? I think this distinction is important, but I'm not sure how to proceed with it.

Reply:
One more thing: `User.created_at` won't work. `.created_at` needs to be invoked on the instance in the loop, not the class here.

But within the loop if we can determine which users _tried_ to accept an invitation that would definitely be the thing to check for.

If `invitation_accepted_at` is properly being set in that path, we should definitely check for users who _do_ have an `invitation_accepted_at` but do not have a `registered_at`.

Reply:
.... And technically setting `user.registered_at = user.invitation_accepted_at` would be the most accurate date for that value.